routes/activateRoute.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
```

Comment:
Since activating a user has to do with a user, this route can still sit in the `userRoutes`